### PR TITLE
Script to support bash auto completion

### PR DESCRIPTION
**Pull request by ganeshmaharaj**
Signed-off-by: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>

### CODE REVIEW

**Comment by mcastelino** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -765,3 +765,18 @@ Removing ccloudvm service
...
 Once you run ccloudvm teardown, ccloudvm will be unusable until you run ccloudvm setup.
+
+## Misc
+
+### bash completion
+The script that might help with auto completion is located [here](./scripts/ccloudvm_bashcomplete). This can be either placed at `/etc/bash_completion.d/` of your machine or as described below.
+
+```
+$ mkdir ~/.bash_completion.d
+$ scp scripts/ccloudvm_bashcomplete ~/.bash_completion.d/ccloudvm
```
Comment (markdryan) [inline README.md]: should this be cp rather than scp?

**Comment by markdryan** [comment]
@ganeshmaharaj this is great thanks.  Could you add a body to the commit message.  Also, I needed to start a new shell before the bash completion worked.  Is this expected?  If so perhaps we should mention this in the documentation.

File: README.md (hunk 1 above)
Comment (ganeshmaharaj) [reply README.md]: Yeah.. should be cp. Or I can make them curl it from the github url. Leaning towards the github url. WDYT?

**Comment by ganeshmaharaj** [comment]
@markdryan yes, will add a body and also add the note that says they will need a new she'll for it to start working.

File: README.md (hunk 1 above)
Comment (markdryan) [reply README.md]: Either is fine for me.  The curl command would be useful if we were to ever do a binary distribution but this is unlikely so cp is fine.

**Pull request merged**